charts/templates/statefulset.yaml

```diff
@@ -23,8 +24,104 @@ spec:
       affinity: {{- toYaml . | nindent 8}}
       {{- end }}
       terminationGracePeriodSeconds: {{ .Values.terminationGracePeriod }}
-      {{- if ne .Values.bootstrapHostName "" }}
+      {{- if or ( ne .Values.bootstrapHostName "") (eq .Values.tls.enableOnDefaultAppServers true ) }}
       initContainers: 
+      {{- if eq .Values.tls.enableOnDefaultAppServers true }}
+      - name: copy-certs
+        image: "finalgene/openssh"
```

Comment:
We should not hardcode the docker image into the statefulset.
It should be part of the value file as customer may need to use private registry.

Reply:
This has been fixed with the updated values file for initContainer section

The new values file will be something like:
initContainers:
  configureGroup:
    image: "curlimages/curl:8.4.0"
    pullPolicy: IfNotPresent
  copyCerts:
    image: "finalgene/openssh:9.1"
    pullPolicy: IfNotPresent

and the code will be:
      - name: copy-certs
        image: {{ .Values.initContainers.copyCerts.image | quote }}
        imagePullPolicy: {{ .Values.initContainers.copyCerts.pullPolicy | quote }}

Please let me know if such change looks good to you.